NodeChrome/Dockerfile

```diff
@@ -46,3 +46,6 @@ COPY generate_config /opt/bin/generate_config
 # Chrome Launch Script Modification
 #=================================
 COPY chrome_launcher.sh /opt/google/chrome/google-chrome
+
```

Comment:
Why do you need to change the Dockerfile? Besides, please read the note on top and only change the .txt file.